ios/Cartfile

```diff
@@ -1,2 +1,4 @@
 github "stephencelis/SQLite.swift" "swift-2.3"
-github "mozilla-magnet/magnet-scanner-ios" ~> 0.2.3
+github "arcturus/magnet-scanner-ios" "geo-scanner"
+github "Alamofire/Alamofire" ~> 3.5.0
+github "SwiftyJSON/SwiftyJSON" "swift2"
```

Comment:
it's the way carthage works, it doesn add subdependencies.

We need then in the other project in order to compile, but what we get is the magnet-scanner-ios framework, we need those 2 libraries in the client to have all the binaries (the 3 of them) that make everything work.